package.json

```diff
@@ -2,48 +2,54 @@
   "name": "sauce-testcafe-runner",
   "version": "0.0.0",
   "description": "Sauce Labs Testcafe runner",
-  "author": "<dev@saucelabs.com>",
+  "author": "<devx@saucelabs.com>",
   "license": "MIT",
   "private": true,
   "bin": "./bin/testcafe",
-  "main": "src/console-wrapper.js",
+  "main": "lib/console-wrapper.js",
   "scripts": {
-    "test": "node src/console-wrapper.js",
-    "lint": "eslint tests/ src/",
+    "build": "tsc",
+    "test": "tsc; node lib/console-wrapper.js",
+    "lint": "eslint tests/ src/**/*.ts",
     "unit-test": "jest --env node"
   },
   "keywords": [
     "testcafe"
   ],
   "dependencies": {
     "@saucelabs/testcomposer": "^1.0.0",
+    "@tsconfig/node18": "^1.0.1",
     "dotenv": "16.0.3",
     "mocha-junit-reporter": "^2.2.0",
     "sauce-testrunner-utils": "1.0.0",
     "testcafe": "2.3.1",
     "testcafe-browser-provider-ios": "0.5.0",
     "testcafe-reporter-saucelabs": "1.0.1",
-    "typescript": "4.9.5",
     "xml-js": "1.6.11",
     "yargs": "17.7.1"
   },
   "devDependencies": {
     "@babel/core": "^7.19.3",
     "@babel/preset-env": "^7.19.4",
+    "@babel/preset-typescript": "^7.21.4",
     "@types/chai": "^4.2.12",
-    "@types/jest": "^29.2.0",
+    "@types/jest": "^29.5.0",
     "@types/jquery": "^3.5.1",
     "@types/mocha": "^8.0.3",
     "@types/node": "^18.15.7",
-    "babel-jest": "^29.1.2",
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "@typescript-eslint/parser": "^5.58.0",
+    "babel-jest": "^29.5.0",
     "eslint": "^7.9.0",
     "eslint-config-appium": "^4.5.0",
-    "eslint-plugin-jest": "^24.1.3",
+    "eslint-plugin-jest": "^24.7.0",
     "eslint-plugin-testcafe": "^0.2.1",
     "husky": "^4.3.0",
-    "jest": "^29.2.2",
+    "jest": "^29.5.0",
     "mocha": "^10.1.0",
-    "release-it": "^15.5.0"
+    "release-it": "^15.5.0",
+    "ts-jest": "^29.1.0",
+    "typescript": "^5.0.4"
```

Comment:
Hm careful with moving this to devDependencies. After all, we were providing this as a runtime dependency to the projects that use this TestCafe runner. This might break projects that relied (for better or worse) on typescript already being present on the VMs.